VirtoCommerce.Storefront/Domain/StaticContent/StaticContentService.cs

```diff
@@ -108,7 +110,24 @@ private void LoadAndRenderContentItem(string contentPath, ContentItem contentIte
 
             try
             {
-                metaHeaders = ReadYamlHeader(content);
+                if (contentPath.EndsWith(".json"))
```

Comment:
Due that fact what the `StaticContentService` is now supports the multiple types of meta/frontmatter headers formats  I suggest to rename `ReadYamlHeader` method to `ReadMeatHeaders` and move this logic there and left this code as it.
But it would be better to define a new strategy for this type of logic and use it through abstraction. And inject in the ctor the list of this strategies or use chain of responsibility pattern
```C#
interface IMetaHeaderLoader 
{
void LoadMetaHeaders(string content,  IDictionary<string, IEnumerable<string>> metaHeaders);
}
```

Reply:
Class `StaticContentLoaderFactory` resolves loaders for different types of pages.
`MarkdownContentLoader` for .md files
`PageBuilderContentLoader` for .page files